README.md

```diff
@@ -8,42 +8,56 @@ A command line tool that searches files within GitHub repositories.
 
 Ebi (えび) is [Japanese for prawn/shrimp](https://translate.google.com/#view=home&op=translate&sl=en&tl=ja&text=Prawn), and intends to be a small little tool to crawl through your sea of code on GitHub, finding you nuggets of information.
 
+## Global installation (recommmended)
+
+`npm install --global @financial-times/ebi`
+When you run the tool, it will automatically notify you if there is a newer version of it available for you to update to.
+
+[You can disable notifications](https://www.npmjs.com/package/update-notifier#user-settings) if you'd prefer not to be notified about updates.
+
+## No installation
+
+`npx @financial-times/ebi`
+The npx command lets you use this tool without installing it. However, each time you use npx it downloads the whole package from the npm registry, which takes a while. That's why global installation is reccommended.
```

Comment:
You'll want an extra newline, so the code doesn't go on the same line as the description

```suggestion

The npx command lets you use this tool without installing it. However, each time you use npx it downloads the whole package from the npm registry, which takes a while. That's why global installation is reccommended.
```